frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/base/cover_title_command.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:appflowy/plugins/document/presentation/editor_plugins/header/cover_title.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+
+/// Press the backspace at the first position of first line to go to the title
+///
+/// - support
+///   - desktop
+///   - web
+///
+final CommandShortcutEvent backspaceToTitle = CommandShortcutEvent(
+  key: 'backspace to title',
+  command: 'backspace',
+  getDescription: () => 'backspace to title',
+  handler: (editorState) => _backspaceToTitle(
+    editorState: editorState,
+  ),
+);
+
+KeyEventResult _backspaceToTitle({
```

Comment:
Shouldn't it also bring the paragraph up to the title?

Reply:
Yes. But I'm struggling with the reminder/date/linked page issue. These texts were formatted with $, it makes no sense to convert them back to $ when bringing the paragraph to title. :(